uint128.go

```diff
@@ -47,14 +47,7 @@ func (u uint128) addOne() uint128 {
 }
 
 func u64CommonPrefixLen(a, b uint64) uint8 {
-	for i := uint8(0); i < 64; i++ {
-		if a == b {
-			return 64 - i
-		}
-		a >>= 1
-		b >>= 1
-	}
-	return 0
+	return uint8(bits.LeadingZeros64(a ^ b))
```

Comment:
gahh! I knew this trick and forgot it when I wrote this code. Thanks!

Reply:
Last night I re-derived this from first principles, then promptly forgot to implement it. Thanks Josh!